bypass_url_parser.py

```diff
@@ -408,11 +399,11 @@ def _run_curl(self, item):
                     result = result.replace(os.linesep, "\n")
 
                     # Store command result in CurlItem object
-                    if not self.proxy:
-                        item.response_raw_output = result
-                    else:
+                    if self.proxy and "HTTP/1.0 200 Connection established" in result:
```

Comment:
I'm not sure we want to hide this, a difference in the response line would also mean that the server works as an outbound proxy, which is a really interesting information !

Is this header from the `-x` curl option or for the test of the `CONNECT` header? 
I'm not sure, but it's worth keeping when we use the CONNECT or similar header, to detect working prroxies! ^.^

Reply:
Noted point but no link with the `CONNECT` method or proxy detection. This fix resolves a bug in response parsing when a proxy is set (`-x http://127.0.0.1:8080`)